observability-test/database.ts

```diff
@@ -975,9 +983,10 @@ describe('Database', () => {
     });
 
     it('with no errors', done => {
-      database.getTransaction((err, transaction) => {
+      database.getTransaction(async (err, transaction) => {
```

Comment:
```suggestion
      database.getTransaction((err, transaction) => {
```

Reply:
done, thank you @alkatrivedi! Kindly help me re-run the bots!

Reply:
done.